pt-client-2/pt-client-2/pt_subdev_config.h

```diff
@@ -0,0 +1,12 @@
+
+#ifndef __PT_SUBDEV_CONFIG_H__
+#define __PT_SUBDEV_CONFIG_H__
+
+#define COMPONENT_NAME_STR "MAIN"
```

Comment:
Shouldn't the component name and version come from PT?

Reply:
The pt-client-2 is used by PT only. The pt-client-2 has apis that is used by PT only.